scripts/build_ssg_content.sh

```diff
@@ -20,7 +20,7 @@ rm -rf "$BUILD_DIR"
 
 # Clone the repo and checkout the latest tag
 git clone "$REPO" "$BUILD_DIR" && pushd "$BUILD_DIR"
-TAG="$(git describe --abbrev=0 --tags)"
+TAG="$(git tag | tail -1)"
```

Comment:
This isn't actually determinative of the most recent tag. This will be impacted by the git config and default sort orders. The prior command _should_ work anywhere. Was it erroring or returning the wrong value? Maybe new git versions are behaving differently...

Reply:
The original code wasn't returning the latest tag.  It would return an older version.  I'm using version `2.26.0`

Reply:
Oh geez, they have two branches now. Have since v0.1.48. The current tags are on the `stable` branch. The prior command retrieved the latest tag only for the current branch. To get the latest across _all_ branches:

```suggestion
TAG="$(git describe --tags $(git rev-list --tags --max-count=1))"
```

Reply:
Ah.  I didn't catch the second branch.  Okay.  I updated that line per your suggestion.